src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+
+    if (inversedRobot[newKey]) {
```

Comment:
checklist 3. use correct check for property presence in object. Some properties can be present, but still contain falsy value.